### PR TITLE
seven-kingdoms 2.15.5 (new formula)

### PR DESCRIPTION
**Pull request by wolf81**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by wolf81** [comment]
Previous discussion is here: https://github.com/Homebrew/homebrew-core/pull/113203

Would be nice if we could have an alias `7kaa` which would be the same name as the binary.

**Comment by SMillerDev** [comment]
Why create a new pull request? This is much harder to review now.

**Comment by SMillerDev** [comment]
The alias can be added in the `Aliases` directory of the repo.

**Comment by wolf81** [comment]
> Why create a new pull request? This is much harder to review now.

I thought it was needed to rename the formula. When I tried to merge the rename, the building process failed.

File: Aliases/7kaa.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class SevenKingdoms < Formula
```
Comment (SMillerDev) [inline Aliases/7kaa.rb:1]: This should be a symlink, not a copy of the file

Comment (wolf81) [reply Aliases/7kaa.rb:1]: ah I see

File: Formula/seven-kingdoms.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  def install
+    system "./configure", *std_configure_args
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    pid = fork { exec bin/"7kaa", "-win", "-demo" }
+    sleep 5
+    system "kill", "-9", pid
```
Comment (chenrui333) [inline Formula/seven-kingdoms.rb:33]: this test can be improved a bit.

**Comment by chenrui333** [comment]
Can you also squash the commits and rebase to the latest master.

**Comment by wolf81** [comment]
> This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

@chenrui333 I've squashed the commits a couple of weeks ago, but still not sure how you'd like to see the test improved. I don't see any way to really improve it, except maybe check for the existence of a binary on the file system.

**Review by chenrui333 — DISMISSED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@wolf81, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**